dotcom-rendering/src/model/decideBadge.ts

```diff
@@ -61,19 +61,3 @@ export const getBadgeFromSeriesTag = (
 			: undefined; // No badge or special badge found
 	}
 };
-
-/**
- * Return a badge based on the series tag or container branding
- *
- * Try to fetch badge using series tag first
- * Otherwise try to fetch badge using branding elements
- * Returns undefined as default / if no matches found
- */
-export const decideBadge = (
```

Comment:
Not used anymore as we're calling directly `decidePaidContentBadge` and `decideEditorialBadge` from `enhanceCollections.ts`